sql/topology/nodeNetwork.sql

```diff
@@ -201,8 +201,12 @@ BEGIN
     vst_line_locate_point := 'st_linelocatepoint';
   END IF;
 
+  create schema if not exists temp_nodenetwork;
+  drop table if exists temp_nodenetwork.intergeom;
+  drop table if exists temp_nodenetwork.inter_loc;
+
   -- First creates temp table with intersection points
-  p_ret = 'CREATE TEMP TABLE intergeom ON COMMIT DROP AS (
+  p_ret = 'CREATE TABLE temp_nodenetwork.intergeom AS (
```

Comment:
The table should be temporary as an extension should not create permanent tables